frontend/App.tsx

```diff
@@ -1,15 +1,17 @@
 import './App.css'
 
-import LoginModal from './components/login-modal'
+import LoginModal from './components/login'
 import Header from './components/Header'
 
 function App () {
   return (
-    <div>
-      <Header/>
+    <div className = "flex flex-col justify-start content-center p-7">
```

Comment:
I think for the App component, we should keep the 'Header' in, and split the login and logout pages into separate pages. Everything else (create new trip, trips dashboard, etc.) would live under 'App', since we want a Header on all those pages.